src/commands/kv/mod.rs

```diff
@@ -119,9 +120,13 @@ fn url_encode_key(key: &str) -> String {
 
 // For handling cases where the API gateway returns errors via HTTP status codes
 // (no KV error code is given).
-fn give_status_code_context(status_code: StatusCode) {
-    if let StatusCode::PAYLOAD_TOO_LARGE = status_code {
-        message::warn("Returned status code 413, Payload Too Large. Make sure your upload is less than 100MB in size")
+fn print_status_code_context(status_code: StatusCode) {
+    match status_code {
+        StatusCode::PAYLOAD_TOO_LARGE =>
```

Comment:
Is the payload in question the one that we upload to the workers api? If so I'm not sure quite how helpful this error messsage is - there isn't really an actionable here that relates to what the user understands wrangler to be doing. "Payload" seems to be an implementation detail.

Reply:
Ideally this error code is never even hit via wrangler--we actually enforce a maximum upload size of 50MB! This is just conservative error handling out of an abundance of caution.

Reply:
Awesome! Could you add a comment saying this :)

Reply:
for sure!